wordpress-mobile/WordPress-iOS
# Use the new create icon for New Post in Quick Actions menu.

Fixes #9199

To test:
Force press the app icon on a 3D Touch-capable device to open the Quick Actions menu. You should see the new icon for "New Post" as seen in the screenshot below.

<img width="714" alt="comparison" src="https://user-images.githubusercontent.com/1396552/59530599-045c6e00-8eb2-11e9-85bc-e97f8fc1012e.png">
